iree/tools/translate_main.cc

```diff
@@ -48,6 +49,7 @@ int main(int argc, char **argv) {
   llvm::InitLLVM y(argc, argv);
 
   mlir::registerMlirDialects();
+  mlir::registerMlirPasses();
```

Comment:
I think `registerMlirPasses` should not be necessary, as `registerAllPasses` isn't called in `mlir-translate`, see 
[here](https://github.com/llvm/llvm-project/blob/549e87f3d04bbae91bc7bc38609ce7073e2c8a6d/mlir/tools/mlir-translate/mlir-translate.cpp#L49).

Reply:
Yup, I can remove from here.